### PR TITLE
[WIP] Batch threaded raftstore

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Used to collect review comments, will be closed after jay finished all the works.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -90,7 +89,8 @@ panic_hook = { path = "components/panic_hook" }
 git = "https://github.com/pingcap/murmur3.git"
 
 [dependencies.rocksdb]
-git = "https://github.com/pingcap/rust-rocksdb.git"
+git = "https://github.com/busyjay/rust-rocksdb.git"
+branch = "hot-fix"
```
Comment (zhangjinpeng1987) [inline Cargo.toml]: ???

Comment (siddontang) [reply Cargo.toml]: PTAL @huachaohuang

**Comment by BusyJay** [comment]
You are using the wrong branch.

File: src/raftstore/store/fsm/transport.rs [hunk 2]
```diff
@@ -0,0 +1,1512 @@
+// Copyright 2018 PingCAP, Inc.
```
Comment (zhangjinpeng1987) [inline src/raftstore/store/fsm/transport.rs]: 2019

**Pull request closed without merging**